catalog/templatetags/query_transform.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
+    updated = request.GET.copy()
+    for key, value in kwargs.items():
+        if value is not None:
```

Comment:
```suggestion
        if value is not None and value != "":
```
lub po prostu
```suggestion
        if not value:
```
Gwarantujesz sobie wtedy większe pokrycie
```